Dmitriy_Vysotskyy2.0/PageObjects/HomePage.cs

```diff
@@ -0,0 +1,90 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.PageObjects;
+using SeleniumExtras.WaitHelpers;
+
+namespace Dmitriy_Vysotskyy2._0.PageObjects;
+
+public class HomePage : BasePage
+{
+    private string _urlLink = "https://www.demoblaze.com";
```

Comment:
This is not correct place for application URL. Let's create class something like ApplicationConfig and put it there. The same for DragAndDrop